app.rb

```diff
@@ -5,14 +5,17 @@
 require_relative 'student'
 require_relative 'teacher'
 require_relative 'validate_input'
+require_relative 'store_data'
+require_relative 'load_data'
+require 'json'
 
 class App
   attr_accessor :books, :people, :rentals
 
   def initialize
-    @books = []
-    @rentals = []
-    @people = []
+    @books = load_books
+    @rentals = load_rentals
```

Comment:
- [ ] Kindly try to handle the case when one or more of the files are missing as per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_preserve_data.md#:~:text=Remember%20to%20handle%20the%20case%20when%20one%20or%20more%20of%20the%20files%20are%20missing.). As now when I remove the `Json` files and try to start the program I get this error.
![image](https://github.com/AjroudiRida/school-library/assets/24703171/b3037156-f8db-46b4-938a-986ce0709787)